packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -2300,14 +2299,31 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
   async buildHashesIndex(entities: Entity[]): Promise<Map<string, string>> {
     const hashMap = new Map<string, string>();
     const pureModelContextData = new V1_PureModelContextData();
+    /**
+     * @ToDelete
+     * This will be deleted once all users have migrated to using full function signature as function name
+     */
+    const TEMPORARY_elementToEntityMap = new Map<
```

Comment:
could the map just be Map<String, String>?
I think all we need is elementPath to entityPath right?